### ISSUE TITLE
Batman: Vengeance (NTSC-U) not loading through SD2SP2

### ISSUE THREAD

**Issue by joaquincoelho** [emukidid/swiss-gc#307]
Batman Vengeance is not working when trying to load it from the SD2SP2, all I get is a blank screen. It works fine when loaded from the from memory slot with the swap trick though.

Tried 3 different memory cards (Samsung, Lexar and generic). Same results. All of them failed with SD2SP2 but worked when loading from the memory card slot.

*Swiss version: v0.4r775-1
*microSD sizes tested: 16GB, 64GB.
*DOL MD5: F54FD223EC22DEF35FE06A0ED4A7CAD9

Thanks.

**Comment by joaquincoelho** [comment]
Tested on v0.4r775-1

**Comment by joaquincoelho** [comment]
microSD sizes tested: 16GB, 64GB.

All of them failed with SD2SP2 but worked when loading from the memory card slot.

**Comment by Extrems** [comment]
This is a OSPanic in EXIDetach with EXI_CHANNEL_2, called upon a EXISelect failure in a old version of HIOEnumDevices with a makeshift EXIGetID.

**Comment by Extrems** [comment]
```
Dolphin OS $Revision: 37 $.
Kernel built : Jul 19 2001 05:42:41
Console Type : Retail 3
Memory 24 MB
Arena : 0x8051f400 - 0x817ffc60
app booted from bootrom
__ARChecksize(): Initializing for RevB+ SDRAM controller...
ARInit(): ARAM size        : 16777216 bytes
ARInit(): USER Base address: 0x00004000
ARInit(): Refresh          : 0x009C
AIInit(): DSP is 32KHz
Initializing AX
Initializing AXAlloc code module
Initializing AXVPB code module
Initializing AXSPB code module
Initializing AXAux code module
Initializing AXCL code module
Initializing AXOut code module
Failed assertion 0 <= chan && chan < 2 in "OSExi.c" on line 748.

Address:      Back Chain    LR Save
0x8051f0a0:   0x8051f130    0x800dadd8
0x8051f130:   0x8051f150    0x800dbcbc
0x8051f150:   0x8051f178    0x800d4b6c
0x8051f178:   0x8051f190    0x801a8ce4
0x8051f190:   0x8051f198    0x8018b5e0
0x8051f198:   0x8051f1a0    0x801a1730
0x8051f1a0:   0x8051f210    0x802e6278
0x8051f210:   0x8051f258    0x80038f8c
0x8051f258:   0x8051f260    0x8012bc84
0x8051f260:   0x8051f268    0x80134d1c
0x8051f268:   0x8051f388    0x80138268
0x8051f388:   0x8051f3c0    0x80137d50
0x8051f3c0:   0x8051f3f0    0x80007030
0x8051f3f0:   0x8051f3f8    0x80007158
0x8051f3f8:   0xffffffff    0x800052d8
```

**Comment by kynrek** [comment]
I tested this with a samsung evo256 and transcend 64gb both freeze at the Ubisoft logo for me. Also using  Swiss v0.4r775-1

**Comment by Extrems** [comment]
This issue was already known for at least a month and has already been debugged as seen above.
Fix pending.

**Comment by Extrems** [comment]
Fixed in 67e4a37 (r776).
